### ISSUE TITLE
What's the result

### ISSUE THREAD

**Issue by secina91** [RBirkeland/MVCNN-PyTorch#5]
May I ask have you re-produced the results in the paper? Many thanks!

**Comment by RBirkeland** [comment]
It's quite some time ago now, but I believe I achieved around 87% validation accuracy fairly quickly (can't remember the epoch exactly), I'm sure that can be raised a bit higher with longer training time and decaying the learning rate.
